app.js

```diff
@@ -0,0 +1,7 @@
+import * as config from './config/taskInfo.json';
+import { Product, User } from 'models';
+
+console.log(config.name);
+
+const product = new Product();
```

Comment:
what are you going to do with these objects?

Reply:
Probably home task creators invent what to do with this later. As for now it's just log a message from constructor